react/FilePath/Readme.md

```diff
@@ -0,0 +1,22 @@
+#### File Path inside a non interactive container
+
+```
+import FilePath from 'cozy-ui/transpiled/react/FilePath';
+
+<FilePath text="/Partages reçus/Cozy 🗄 - Team/Customers & Partners 🛒/Xxxxxx (Xxxxxxxx)/4_Suivi opérationnel/Point de synchro" />
+```
+
+#### File Path inside an interactive container
+
+```
+import FilePath from 'cozy-ui/transpiled/react/FilePath';
+
+<a href="#"><FilePath text="/Partages reçus/Cozy 🗄 - Team/Customers & Partners 🛒/Xxxxxx (Xxxxxxxx)/4_Suivi opérationnel/Point de synchro" /></a>
```

Comment:
```suggestion
<Link href="#"><FilePath text="/Partages reçus/Cozy 🗄 - Team/Customers & Partners 🛒/Xxxxxx (Xxxxxxxx)/4_Suivi opérationnel/Point de synchro" /></Link>
```

Reply:
I was thinking we could show it can be used with traditional anchor, if for some reason someone didn't want a react-router or material Link?

Reply:
`#### File Path inside an anchor

```
import FilePath from 'cozy-ui/transpiled/react/FilePath';

<a href="#">
  <FilePath>/Partages reçus/Cozy 🗄 - Team/Customers & Partners 🛒/Xxxxxx (Xxxxxxxx)/4_Suivi opérationnel/Point de synchro"</FilePath>
</a>
````

With a title?

Reply:
@Crash-- I added a description to show it's working inside an anchor. In my opinion if the FilePath must NEVER be inside an anchor but always inside a link (if clickable), then we should compose it in another component (FilePathLink for instance)

Reply:
Since cozy-ui is a lib based on material-ui, you should always use the components that the lib provide. Since it will be themed and so on. So I really think it's not a good idea to make exemples which do not represent the reality

Reply:
<FilePathLink>